transit_odp/browse/templates/browse/disruptions/download_disruptions.html

```diff
@@ -0,0 +1,75 @@
+{% extends 'browse/base.html' %}
+
+{% load static %}
+{% load i18n %}
+{% load breadcrumbs %}
+
+{% block title_tag %}
+    {% trans 'All fares data' %}
+{% endblock %}
+
+{% block breadcrumb.content %}
+    {{ block.super }}
+    {% breadcrumb_url 'Download All Data' 'downloads' host hosts.data %}
+    {% breadcrumb 'All Fares Data' %}
+{% endblock %}
+
+{% block heading.title.text %}{% trans 'All disruption data' %}{% endblock %}
+
+{% block secondary.title %}
+    <h3 class="govuk-heading-l">{% trans 'Download file' %}</h3>
+    <p class="govuk-body">
+        {%  blocktrans %}
+            Download a snapshot of all live and upcoming disruptions within England
+        {%  endblocktrans %}
+    </p>
+    <ul class="govuk-list">
+        <li>
+            <b>{% trans 'Format' %}:</b>
+            {% trans 'SIRI-SX File' %} {% include "browse/snippets/help_modals/sirisx.html" %}
+        </li>
+        <li><b>{% trans 'File type' %}:</b> {% trans 'ZIP file' %}</li>
+        <li><b>{% trans 'Mode' %}:</b> {% trans 'Bus, Tram, Light Rail' %}</li>
+        <li>
+            <b>{% trans 'Update frequency' %}:</b>
+            {% trans 'Every 5 minutes Coverage: North of England (Greater Manchester, Merseyside, South Yorkshire, North Yorkshire, Nexus) Source: Local Authority' %}
```

Comment:
Can we confirm the update frequency?

Wireframes says 5mins but I remember talking about this at standup and I thought we update frequency is actually 1 min?

Reply:
resolved